server/controllers/cohort/getCohorts.js

```diff
@@ -0,0 +1,17 @@
+const cohorts = require('../../database/models/cohort');
+
+module.exports = (req, res, next) => {
```

Comment:
@alidah0 It's a good idea to declare a named function and then export it on a separate line, instead of directly exporting an anonymous function.  For example:
```
const getCohorts = (req, res, next) => {
  ...
}
module.exports = getCohorts
```
This helps with debugging because then you will see your function name in stack traces if you are debugging errors.